cSpell.json

```diff
@@ -0,0 +1,408 @@
+        "konmaripoweb",
+        "Landwerth",
+        "lifecycles",
+        "likert",
+        "Linq",
+        "Liskov",
+        "livestream",
+        "Lucene",
+        "Maximo",
+        "microservice",
+        "microservices",
+        "Minecraft",
+        "mistersql",
+        "mkdir",
+        "moreso",
+        "msbuild",
+        "msys",
+        "MVC's",
+        "MVPs",
+        "Nagios",
+        "Namecheap",
+        "Namecheap's",
+        "Nerdgasm",
+        "nevermind",
+        "Neward",
+        "Ninject",
+        "nlkl",
+        "nlkl's",
+        "Noura",
+        "Novotny",
+        "NTFS",
+        "ntlm",
+        "Nuget",
+        "nunit",
+        "Octokit",
+        "octopress",
+        "octopusdeploy",
+        "Ooof",
+        "oooh",
+        "openssl",
+        "Opserver",
+        "org's",
+        "Orgs",
+        "ORMs",
+        "Osherove",
+        "OTRS",
+        "parenttitle",
+        "parenturl",
+        "PCIE",
+        "Photoshop",
+        "Pingendo",
+        "Pluralsight",
+        "POCOs",
+        "Powershell's",
+        "Proto",
+        "Pssst",
+        "Rakefile",
+        "refactorings",
+        "Repos",
+        "Resharper",
+        "RFPs",
+        "ridk",
+        "Roadmap",
+        "Rsync",
+        "Saad",
+        "SATA",
+        "screencast",
+        "Sectigo",
+        "Serilog",
+        "servername",
+        "signkey",
+        "skillset",
+        "skillsets",
+        "Smtp",
+        "solutioning",
+        "Sommelier's",
+        "SpecFlow",
+        "SSRS",
+        "Statiq",
+        "Storify",
+        "sysdate",
+        "teamcity",
+        "Tekkit",
+        "templating",
+        "testwithoutbuild",
+        "thumbprint",
+        "toolset",
+        "Touchpad",
+        "Townhall",
+        "Trello",
+        "Triaging",
+        "Tweetdeck",
+        "Twilio",
+        "unanet",
+        "vcodec",
+        "verifypeer",
+        "Visio",
+        "visualsvn",
+        "VM's",
+        "vmware",
+        "VPNing",
+        "vsts",
+        "Walkthrough",
+        "WMATA",
+        "WMATA's",
+        "Womp",
+        "xluh",
+        "yada",
+        "Youngkin",
+        "Yuuup",
+        "Zikmund"
+    ],
+    "ignoreWords": [
+        "dcca",
+        "sublicense",
+        "MERCHANTABILITY",
+        "NONINFRINGEMENT",
+        "StackOverflow",
+        "devrel",
+        "Grigorik",
+        "Icaza",
+        "Ilya",
+        "de",
+        "roadmap",
+        "phpdoc",
+        "nav",
+        "SDK",
+        "jumpstart",
+        "SemVer",
+        "APIs",
+        "Analytics",
+        "convension",
+        "DevEx",
+        "triaged",
+        "Jira",
+        "api",
+        "DevRel",
+        "MVPs",
+        "ai's",
+        "Chrastina",
+        "Ondřej",
+        "Ondrej",
+        "Polesny",
+        "Kentico",
+        "Palas",
+        "Petr",
+        "Ilesh",
+        "Mistry",
+        "McKeiver",
+        "Polesny's",
```

Comment:
Is it necessary to add derivations like this? I would expect the `Cspell` to know that if `Polesny` is a valid word then `Polesny's` if fine as well.

Reply:
I copied them from the first run

Reply:
I have updated the list, it should be gone now